tests/reference/asr-assert1-1ce92ea.stderr

```diff
@@ -0,0 +1,15 @@
+Internal Compiler Error: Unhandled exception
+Traceback (most recent call last):
+[2m  Binary file "[0m[1m[35m$DIR/src/bin/lpython[39m[0m[2m", in _start()[0m
+[2m  File "[0m[1m[35m/build/glibc-S9d2JN/glibc-2.27/csu/../csu/libc-start.c[39m[0m[2m", line 310, in __libc_start_main()[0m
+[2m  File "[0m[1m[35m$DIR/src/bin/lfortran.cpp[39m[0m[2m", line 1386, in main()[0m
+    } else if (arg_backend == "cpp") {
+[2m  File "[0m[1m[35m$DIR/src/bin/lfortran.cpp[39m[0m[2m", line 529, in emit_asr()[0m
+    return 0;
+[2m  File "[0m[1m[35m$DIR/src/lfortran/python_serialization.cpp[39m[0m[2m", line 44, in LFortran::Python::deserialize_ast(Allocator&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)[0m
+    return v.deserialize_node();
+[2m  File "[0m[1m[35m$DIR/src/lfortran/python_ast.h[39m[0m[2m", line 6565, in LFortran::Python::AST::DeserializationBaseVisitor<LFortran::Python::ASTDeserializationVisitor>::deserialize_node()[0m
+    uint8_t t = self().read_int8();
+[2m  File "[0m[1m[35m$DIR/src/libasr/bwriter.h[39m[0m[2m", line 201, in LFortran::TextReader::read_int8()[0m
+    uint64_t n = read_int64();
+LFortranException: read_int64: Expected integer, got `d`
```

Comment:
We are updating `./run_tests` is to verify that the implementation is working correctly, and none other changes would break the current implementation. Then we use this reference to test in the CI (Actions).

As can be seen, this updated one has an error that shouldn't be pushed...

Reply:
Removed.

Reply:
Before updating the `tests` make sure that there are no errors!

Reply:
I mistakenly ran without updating lpython. So fixed them now.